README.md

```diff
@@ -71,11 +71,12 @@ As fluent-plugin-datadog is an output_buffer, you can set all output_buffer prop
 | **include_tag_key** | Automatically include tags in the record. | false |
 | **tag_key** | Name of the tag attribute, if they are included. | "tag" |
 | **use_ssl** | If true, the agent initializes a secure connection to Datadog. In clear TCP otherwise. | true |
+| **ssl_port** | Port used to send logs over a SSL encripted connection to Datadog (use 443 for the EU region) | 10516 |
 | **max_retries** | The number of retries before the output plugin stops. Set to -1 for unlimited retries | -1 |
 | **dd_source** | This tells Datadog what integration it is | nil |
 | **dd_sourcecategory** | Multiple value attribute. Can be used to refine the source attribtue | nil |
 | **dd_tags** | Custom tags with the following format "key1:value1, key2:value2" | nil |
-| **port** | Proxy port when logs are not directly forwarded to Datadog | 10516 |
+| **port** | Proxy port when logs are not directly forwarded to Datadog and ssl is not used | 10514 |
```

Comment:
Wondering whether we should keep only on port settings that used for both transports and we add default values depending on `use_ssl` ; this would simplify the life of users I think.

Reply:
That could definitely make sense and would be a much easier setup.

Let's open this topic in a different thread and see how we can make sure everything remains backward compatible.